Services/AbstractExporter.php

```diff
@@ -64,27 +65,19 @@ public function export(array $criteria = [], array $options = []): void
      * Get datetime based on user's date format.
      *
      * @param  string|null $value
```

Comment:
Should we add the new parameter `$default` here?

Reply:
Well, the coding standards don't require it, so I've removed `@param  string|null $value` too. The coding standards require the `@return` annotation.